doc/source/flexibility/compressed_columns.rst

```diff
@@ -92,6 +92,27 @@ table is of the data dictionary kind, concurrent reads are allowed, but writes
 are serialized, and reads are blocked by writes. Table read through old read
 views are not supported, similar to |InnoDB| internal DDL transactions.
 
+Interaction with :variable:`innodb_force_recovery` variable
+-----------------------------------------------------------
+
+Compression dictionary operations are treated like DDL operations (The reason
```

Comment:
Compression dictionary operations are treated like DDL operations with the exception of innodb_force_value == 3: with values less than 3, compression dictionary operations are allowed, and with values >= 3, they are forbidden

Reply:
Besides "treated like DDL operations", no further discussion of DDL operations themselves is not needed